dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/test/TestLoadRestorativeWithLimits.cpp

```diff
@@ -287,17 +291,27 @@ TEST(ModelsLoadRestorativeWithLimits, ModelLoadRestorativeWithLimitsContinuousAn
   ASSERT_NO_THROW(modelLoad->evalZ(0));
   ASSERT_NO_THROW(modelLoad->evalMode(0));
   ASSERT_NO_THROW(modelLoad->setGequations());
+  ASSERT_NO_THROW(modelLoad->evalDynamicYType());
+  ASSERT_EQ(yType[0], ALGEBRAIC);
```

Comment:
Is there a possibility to get the DIFFERENTIAL type in the tests?

Reply:
done